firebase-sessions/src/test/kotlin/com/google/firebase/sessions/api/FirebaseSessionsDependenciesTest.kt

```diff
@@ -19,7 +19,7 @@ package com.google.firebase.sessions.api
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.common.truth.Truth.assertThat
 import com.google.firebase.sessions.api.SessionSubscriber.Name.CRASHLYTICS
-import com.google.firebase.sessions.api.SessionSubscriber.Name.PERFORMANCE
+import com.google.firebase.sessions.api.SessionSubscriber.Name.MATT_SAYS_HI
```

Comment:
Is this intended to be left in?

Reply:
This was intentional. We removed perf integration for now to cut scope.